script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -162,22 +181,22 @@ Even from seeing her back alone, I can clearly grasp the extent of the hostile a
 [sha:488174490ed72d24341767dadd8a8b4f3b0f6521]{
 -- Page 286, Offset 17271.
 -- するつもりだ、とは言えない。
-I couldn't finish my sentence.
+――planning on doing, I start to ask, but fall silent.
 }
 [sha:5c6ecb5e1c7f1ff81cbdf29ab0d774dc99b53bee]{
 -- Page 286, Offset 17272.
 -- 彼女が何をしようとしているのか、言葉にするまでもなく判りきっている。
-There was no need for her to put it into words for me to understand what she was about to do.
+I know what she's about to do even without hearing it put into words.
 }
 [sha:63da4b06040e0c7748b8cf8894a9994fea4fcdde]{
 -- Page 286, Offset 17273.
 -- アルクェイドから放たれているのは一点の濁りもない『殺意』だった。
-The thing emanating from Arcueid was, without a shadow of a doubt, her 'killing intent'.
+What emanates from Arcueid right now is an unmistakable 'killing intent'.
```

Comment:
Not so sure about 'killing intent'. Could be 'intent to kill', but I think I might just favor 'murderous intent'.